datadog/resource_datadog_dashboard.go

```diff
@@ -3023,6 +3044,12 @@ func buildTerraformTimeseriesDefinition(datadogDefinition datadog.TimeseriesDefi
 	if datadogDefinition.Markers != nil {
 		terraformDefinition["marker"] = buildTerraformWidgetMarkers(&datadogDefinition.Markers)
 	}
+	if datadogDefinition.Events != nil {
+		terraformDefinition["event"] = buildTerraformWidgetEvents(&datadogDefinition.Events)
+	}
+	if datadogDefinition.Yaxis != nil {
+		terraformDefinition["yaxis"] = buildTerraformWidgetAxis(*datadogDefinition.Yaxis)
```

Comment:
`buildTerraformWidgetAxis` function returns a `map[string]interface{}`, the type of `yaxis` is `TypeList`, so you want to turn it to `[]map[string]interface{}` (you can take a look at the `buildTerraformScatterplotDefinition` function that does exactly what you need 😉)